longestCommonPrefix.py

```diff
@@ -0,0 +1,23 @@
+def f(strs):
+    if strs==[]:
+        return ""
+    else:
+        n=min(map(len,strs))
+        print(n)
+        if n==0:
+            return ""
+        else:
+            prefix=""
+            i=0
+            while len(set([a[i] for a in strs]))==1 and i<n:

```

Comment:
should use instead.
```python
            while i<n and len(set([a[i] for a in strs]))==1:
```
to make sure i < n before checking a[i]

Reply:
I see. Thanks very much!